ipa-core/src/protocol/prss/mod.rs

```diff
@@ -209,15 +180,15 @@ impl<'a, Z: ArrayLength> Iterator for ChunksIter<'a, Z> {
 /// in APIs that expect `Rng`.
 pub struct SequentialSharedRandomness {
     generator: Generator,
-    counter: u128,
+    counter: PrssIndex,
```

Comment:
If you do as I suggest above, then you should not use `PrssIndex` here.  It is better to start with 0_u128, I think.

Reply:
yea, I can do that. The reason I made this change is that `PrssIndex` is used everywhere in `SharedRandomness` trait and we were going that direction (from u128 to `PrssIndex`). @andyleiserson do you have any thoughts here?

Reply:
I'm ok with having PrssIndex128 be the place we enforce limits, but the way in which we do that needs to be clear.

Reply:
Let me know if it is still not clear in the latest version of this PR.